### PR TITLE
add tooltip scale from corner

### PR DESCRIPTION
**Pull request by hiepvo**

### CODE REVIEW

File: assets/html/evolution/tooltip_scale_from_corner.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  <title>{{NAME OF YOUR COMPONENT}}</title>
+  <meta name="description" content="">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Roboto+Slab|Roboto:300,400,700" rel="stylesheet">
+  <link href="../../styles/main.css" rel="stylesheet">
+</head>
+<body>
+<button class="btn btn-primary tt-scale">Projects
+  <span class="tt-scale__vertical"><div class="tt-scale__content">Lorem ipsum, consectetur adipiscing elit. Donec metus orci. faucibus sem nec. Quisque tempus pretium dolor vitae tincidunt.</div>
```
Comment (Gruximillian) [inline assets/html/evolution/tooltip_scale_from_corner.html]: @hiepvo 
Placing `div` inside a `span` is invalid markup. Please change that. Span is an inline element and can hold only other inline elements, div is a block level element and it is not valid to place it inside an inline element regardless if you set its display property to inline with css.

**Comment by Gruximillian** [comment]
@hiepvo 
Please use our [naming conventions](https://github.com/BovAcademy-opensource/evolution-ui/blob/development/.github/CONTRIBUTING.md#naming-conventions) in your scss files. You don't have to write scss or to use our sass mixins, but you need to follow the class naming convention.
After you make the noted changes, make a push to your feature branch and this pull request will be updated. Then let me know so I can review it and hopefully merge it.

**Comment by Gruximillian** [comment]
@hiepvo 
One more thing. Please fill out these placeholders in your html file with appropriate data:
`{{YOUR NAME HERE}}` and
`{{NAME OF YOUR COMPONENT}}`

**Comment by hiepvo** [comment]
thanks @Gruximillian, I'll make  changes on it.

**Comment by hiepvo** [comment]
@Gruximillian , I've made change, Can you review it? Let me know if it need to be changed on something

**Pull request merged**